packages/ui/stories/molecules/Button.stories.tsx

```diff
@@ -33,31 +33,63 @@ storiesOf('ui/molecules/Button', module).add(
   },
   {
     argTypes: {
-      title: {
-        type: 'string',
-        defaultValue: 'Press me',
-        control: {type: 'text'},
-      },
       color: {
         options: Object.keys(lightTheme.colors),
```

Comment:
We should maybe remove colors which are directly a text and not a Color object. This has been done on InfoBubble story I think